dist/styles/angular-patternfly.css

```diff
@@ -171,3 +171,95 @@
 .data-list-pf .row-column {
   padding-right: 5px;
 }
+
+.tiles-view-pf {
+  overflow: auto;
+  padding-top: 20px;
+}
+
+.tiles-view-pf .tile {
+  -webkit-box-shadow: 0 1px 1px rgba(0, 0, 0, .175);
+  background: #fff;
+  border: 1px solid #eee;
+  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.175);
+  display: block;
+  float: left;
+  margin-right: 20px;
+  margin-bottom: 20px;
+  padding: 10px;
+  position: relative;
+  text-align: center;
+}
+
+.tiles-view-pf .tile-check-box {
+  left: 10px;
+  position: absolute;
+  top: 8px;
+  width: 20px;
+  z-index: 1000;
```

Comment:
The idea is only to keep the checkbox on top of the content so that clicking the checkbox does not call selectItem which could be used by the outer code to do something unintended by the user (such as drill down). I will change it to 3.